### ISSUE TITLE
Can we get an english version?

### ISSUE THREAD

**Issue by olufemioyedepo** [GeekQiaQia/vue3.0-template-admin#28]
Hi GeekQiaQia,

Great job for coming up with this template but can we  get an English version?

**Comment by GeekQiaQia** [comment]
> Hi GeekQiaQia,
> 
> Great job for coming up with this template but can we get an English version?

yeah,of course,  it's based on Element Plus UI ,and also you can switch lang from  CH to EN ,  you are welcomed to pr
